tests/headers/bitfield_align.h

```diff
@@ -37,5 +37,27 @@ struct Date2 {
    unsigned short nMonthDay : 6;    // 0..31  (6 bits)
    unsigned short nMonth    : 5;    // 0..12  (5 bits)
    unsigned short nYear     : 8;    // 0..100 (8 bits)
-   unsigned char byte;
+   unsigned char byte : 8;
 };
+
+enum MyEnum {
+    ONE,
+    TWO,
+    THREE,
+    FOUR
+};
+
+struct TaggedPtr {
```

Comment:
Would really prefer to put the new definitions in a new test, because it is **much** easier to debug failing tests when they are small and isolated.

`bitfield_align_2.h`? `issue_726.h`?